server/vcr-server/agent_webhooks/views.py

```diff
@@ -441,6 +441,10 @@ def handle_register_issuer(message):
     issuer_manager = IssuerManager()
     updated = issuer_manager.register_issuer(message)
 
+    # reset the global CredentialManager instance (to clear the CredentialType cache)
+    global credential_manager
```

Comment:
Would it make sense to not use global, and instead initialize the `CredentialManager` in each handler?

Reply:
Does this lose the benefit of caching the credential types?